prebuilt_forms/js/species_details_2.js

```diff
@@ -1,16 +1,29 @@
 jQuery(document).ready(function($) {
 
+  // Adjust CSS on some form elements
+  $('#ctrl-wrap-occurrence-taxa_taxon_list_id').css('margin-bottom', 0)
```

Comment:
The CSS changes should be done by adding a file prebuilt_forms/css/species_details_2.css (you will need to clear Drupal cache for it to be picked up).

Reply:
Sorry - I hadn't realised that.